lib/activerecord-import/import.rb

```diff
@@ -412,7 +412,7 @@ def import_helper( *args )
           column_names = self.column_names.dup
         end
 
-        if models.first.id.nil? && column_names.include?(primary_key) && columns_hash[primary_key].type == :uuid
+        if models.first.id.nil? && column_names.include?(primary_key) && [:integer, :uuid].include?(columns_hash[primary_key].type)
```

Comment:
Hey @mathieujobin, thanks for catching this. I think this wasn't an issue with MySQL because MySQL treats a a `NULL` value in an INSERT statement as meaning "auto-increment please", and PostgreSQL doesn't.

Given the original condition above are you inserting some rows that _do_ have values for the `id` (primary key) column and some that do not? Or does all of your data not have values for `id`?